src/compose/ComposeBox.js

```diff
@@ -184,8 +190,15 @@ class ComposeBox extends PureComponent<Props, State> {
     dispatch(draftUpdate(narrow, message));
   };
 
-  handleMessageAutocomplete = (message: string) => {
+  // See JSDoc on 'onAutocomplete' in 'AutocompleteView.js'.
+  handleMessageAutocomplete = (message: string, completion: string, lastWordPrefix: string) => {
     this.setMessageInputValue(message);
+
+    if (lastWordPrefix === '@') {
+      if (this.mentionWarnings.current) {
+        this.mentionWarnings.current.getWrappedInstance().handleMentionSubscribedCheck(completion);
```

Comment:
This pattern can be shortened with `?.`:
`this.mentionWarnings.current?.getWrappedInstance()?.handleMentionSubscribedCheck(completion)`.

Reply:
I'm getting this flow error:
```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ src/compose/ComposeBox.js:196:7

Flow does not yet support method or property calls in optional chains.

     193│     this.setMessageInputValue(message);
     194│
     195│     if (lastWordPrefix === '@') {
     196│       this.mentionWarnings.current?.getWrappedInstance()?.handleMentionSubscribedCheck(completion);
     197│     }
     198│   };
     199│


Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ src/compose/ComposeBox.js:196:7

Flow does not yet support method or property calls in optional chains.

     193│     this.setMessageInputValue(message);
     194│
     195│     if (lastWordPrefix === '@') {
     196│       this.mentionWarnings.current?.getWrappedInstance()?.handleMentionSubscribedCheck(completion);
     197│     }
     198│   };
     199│



Found 2 errors
```

Reply:
Dang, so be it then.

It looks like this is https://github.com/facebook/flow/issues/4303 ... which, good news, is closed as fixed! In v0.112.0, looks like. Which means we'll get that when we upgrade to RN v0.62: #3782.